test/timezone_names.jl

```diff
@@ -0,0 +1,7 @@
+import TimeZones: timezone_names
+
+names = timezone_names()
+
+@test length(names) >= 429
```

Comment:
I guess this is a good way to track the addition of new timezones :) Didn't North Korea just create their own? i wonder when that will be added.